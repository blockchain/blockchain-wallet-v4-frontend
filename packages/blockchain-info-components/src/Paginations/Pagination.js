import React from 'react'
import styled from 'styled-components'

const Wrapper = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: flex-start;
  align-items: flex-start;
  width: 100%;
  margin: 10px 0;
`

const Pagination = props => {
  const { children } = props
  return <Wrapper>{children}</Wrapper>
}

export default Pagination
