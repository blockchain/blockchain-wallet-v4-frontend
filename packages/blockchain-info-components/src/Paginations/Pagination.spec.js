import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import Pagination from './Pagination'

describe('Pagination component', () => {
  it('default renders correctly', () => {
    const component = shallow(<Pagination selected />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
