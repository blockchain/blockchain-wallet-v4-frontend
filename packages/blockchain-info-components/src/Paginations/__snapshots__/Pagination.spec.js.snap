// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination component default renders correctly 1`] = `<Pagination__Wrapper />`;
